SA/design_concept/content/poc/web_app.typ

```diff
@@ -0,0 +1,12 @@
+= Web Application
+A web application has a frontend and a backend.
+The frontend is served to the user in a web browser and communicates with the backend, which is hosted on a server, via a REST API.
+Depending on the expected load generated by the frontend, the backend can be horizontally scaled, in which case that frontend communicates with the backend via a load balancer.
```

Comment:
```suggestion
```

Do we need this line?

Reply:
Not really, I just ran out of things to write 😆

Reply:
I took it out